mapdata/WarcraftLegacies/AbilityData/Skin/1479749953.json

```diff
@@ -29,7 +25,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Reduces the damage taken from Piercing attacks to \u003CA13X,DataA1,%\u003E%."
+      "Value": "Reduces the damage taken from Magic attacks by \u003CA13X,DataE1,%\u003E%."
```

Comment:
I believe this tooltip is incorrect. The ability you have reduces magic damage _to_ 80%, not by.

Reply:
ty changed